k6-extension/models/tracetestRun.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+import "encoding/json"
+
+type Test struct {
+	ID   string
+	Name string
+}
+
+type TestRun struct {
+	ID      string
+	TraceID string
+}
+
+type TracetestRun struct {
+	Test    Test
+	TestRun TestRun
+}
+
+func NewRun(cliResponse string) *TracetestRun {
+	var tracetestRun TracetestRun
+	json.Unmarshal([]byte(cliResponse), &tracetestRun)
```

Comment:
We are not handling errors here. Is there any chance of `cliResponse` receiving a malformed JSON?

Reply:
Checking 👀